sections/main-product.liquid

```diff
@@ -310,8 +349,34 @@
                           value="{{ variant.id }}"
                         >
                           {{ variant.title }}
-                          {%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% endif %}
                           - {{ variant.price | money | strip_html }}
+                          {%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% endif %}
+                          {%- for block in section.blocks -%}
+                            {%- if block.type == 'inventory' -%}
+                              {%- if variant.available and variant.inventory_management == 'shopify' -%}
+                                {{- ' - ' -}}
+                                {%- if variant.inventory_quantity != 0 -%}
+                                  {%- if block.settings.show_inventory_quantity -%}
+                                    {%- if variant.inventory_quantity <= block.settings.inventory_threshold -%}
+                                      {{- 'products.product.inventory_low_stock_show_count' | t: quantity: variant.inventory_quantity -}}
+                                    {%- else -%}
+                                      {{- 'products.product.inventory_in_stock_show_count' | t: quantity: variant.inventory_quantity -}}
+                                    {%- endif -%}
+                                  {%- else -%}
+                                    {%- if variant.inventory_quantity <= block.settings.inventory_threshold -%}
+                                      {{- 'products.product.inventory_low_stock' | t -}}
+                                    {%- else -%}
+                                      {{- 'products.product.inventory_in_stock' | t -}}
+                                    {%- endif -%}
+                                  {%- endif -%}
+                                {%- else -%}
+                                  {%- if variant.inventory_policy == 'continue' -%}
+                                    {{- 'products.product.inventory_in_stock' | t -}}
+                                  {%- endif -%}
+                                {%- endif -%}
+                              {%- endif -%}
+                            {%- endif -%}
+                          {%- endfor -%}
```

Comment:
Yeah I think this is probably overkill. I'd say if it's no JS then we just don't show anything to avoid having to loop through blocks and having to re run so much logic. 

What do you think @metamoni ?